### DISCUSSION TITLE
Individual cell classes

### DISCUSSION THREAD

**Discussion by mjelle** [fiduswriter/Simple-DataTables#335] [Q&A] [answered]
Is it possible to have individual classes on different rows of a column? I have a scenario where the background color of a cell should be set based on a value in the cell. All original classes are wiped on init, the cellClass only seems to work for the whole column, and i can't get a data attribute set on the dom like this to work `<td data-class="{{$class}}">` Am i missing something?

**Comment by johanneswilm** [comment]
Please take a look at the demos.

**Comment by mjelle** [comment]
Ah got it, found this https://fiduswriter.github.io/simple-datatables/demos/4-render-column-cell/
Either a span inside the td or a conditioned class on the tr:
`rowRender: (row, tr, _index) => {...}`

I can work with that. 

Thanks!

**Comment by johanneswilm** [comment, marked as answer]
Thanks to @SandroHc in version 9.0.0 it is now possible to directly have attributes attached to individual cells. See https://fiduswriter.github.io/simple-datatables/demos/25-cell-attributes/
